### ISSUE TITLE
Unable to distinguish between landmarks of the same type.

### ISSUE THREAD

**Issue by JamieKnightBBC** [bbc/gef-docs#17]
The Promos page: https://bbc.github.io/code-gel/components/promos/ makes extensive use of landmarks.

However landmarks of the same type cannot be distinguished from each other in the context of the landmarks menu in Voiceover. This makes navigating by landmark difficult.

Various documents and advice^ recommends giving landmarks of the same time a label in order to aid navigation.

![vo landmarks list](https://user-images.githubusercontent.com/18007258/47919322-046da780-dea7-11e8-930b-66f5d8abbe02.jpg)

^ https://www.w3.org/TR/WCAG20-TECHS/ARIA11.html, https://fae.disability.illinois.edu/rulesets/LANDMARK_17/, https://www.w3.org/TR/wai-aria-practices/examples/landmarks/complementary.html)
